### ISSUE TITLE
Can it support running directly on Linux systems to limit GPU resources in Docker? Is it still necessary to use k8s?

### ISSUE THREAD

**Issue by GuoYingLong** [NTHU-LSALAB/Gemini#11]
Can it support running directly on Linux systems to limit GPU resources in Docker? Is it still necessary to use k8s?

**Comment by kerwenwwer** [comment]
Yes, using the code base from the master branch or v1.1-bare-metal and following instructions in README.md

**Comment by GuoYingLong** [comment]
@kerwenwwer Do you have a detailed tutorial on how to use it?
